### ISSUE TITLE
Poloniex error: unable to buy, "Not enough BTC."

### ISSUE THREAD

**Issue by carlos8f** [DeviaVir/zenbot#169]
```
{ Error: unable to buy
    at /Users/morpheus.5250/projects/zenbot/extensions/poloniex/exchange.js:173:17
    at Request._callback (/Users/morpheus.5250/projects/zenbot/node_modules/poloniex.js/lib/poloniex.js:79:7)
    at Request.self.callback (/Users/morpheus.5250/projects/zenbot/node_modules/request/request.js:122:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:194:7)
    at Request.<anonymous> (/Users/morpheus.5250/projects/zenbot/node_modules/request/request.js:888:14)
    at emitOne (events.js:101:20)
    at Request.emit (events.js:191:7)
    at IncomingMessage.<anonymous> (/Users/morpheus.5250/projects/zenbot/node_modules/request/request.js:839:12)
    at emitNone (events.js:91:20)
  body: { error: 'Not enough BTC.' },
  desc: 'could not execute buy: error placing order' }
```

**Comment by carlos8f** [comment]
not sure why, I had enough BTC, need memory dumps to debug this.

**Comment by carlos8f** [comment]
fixed by a036999fadc02e584bb73a9153ec81bad0bc2f51
